syntax = "proto3";

package t_catalogue_proto;

import "svg.proto";

message Map {
    double width = 1;
    double height = 2;
    double padding = 3;
    double line_width = 4;
    double stop_radius = 5;
    int32 bus_label_font_size = 6;
    int32 stop_label_font_size = 7;
    repeated double bus_label_offset = 8;
    repeated double stop_label_offset = 9;
    double underlayer_width = 10;
    Color underlayer_color = 11;
    repeated Color color_palette_ = 12;
}